src/main/pg/mco.h

```diff
@@ -28,6 +28,8 @@
 
 typedef struct mcoConfig_s {
     uint8_t enabled[2];
+    uint8_t source[2];
```

Comment:
Are there platforms that support configurable source / divider for both MCOs, or why are these arrays?

Reply:
There is a mux that selects a clock from several sources to the MCO pins.  If I read this right, this interface now supports selecting the mux source as well as enable/disable for MCO1 and/or MCO2.  The 27Mhz clock is created by using an otherwise unused PLL (that is hardcoded to 27MHz? too late to check.) and may not yet have an interface to modify the PLL registers.   As this is a seldom used feature, it probably has not been a high priority.

I am fine rerouting to use MCO1 pin, but I think I might have used MCO2 because MCO1 had a timer option for the pin that I could use for Motor5 or Motor6.